### PR TITLE
Feature/77127 style move buttons

### PR DESCRIPTION
**Pull request by jesseeproductions**
_Ref:_ [C#77127](https://central.tri.be/issues/77127)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:54](https://github.com/moderntribe/the-events-calendar/blob/5f81a2ce9c0561517449b68b8ef8106a9e73c865/src/views/day/single-event.php#L54)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/5f81a2ce9c0561517449b68b8ef8106a9e73c865/src/views/list/single-event.php#L64)

> via jenkins/codesniffer-phpcs

File: src/views/day/single-event.php [hunk 1]
```diff
@@ -58,6 +47,18 @@
 	<?php endif; ?>
 
 </div><!-- .tribe-events-event-meta -->
+
+<!-- Event Cost -->
```
Comment (borkweb) [inline src/views/day/single-event.php]: You'll need to add a `@version 4.4.whatever` line in this file's docblock at the top of the file

File: src/views/list/single-event.php [hunk 2]
```diff
@@ -72,6 +57,22 @@
 
 	</div>
 </div><!-- .tribe-events-event-meta -->
+
+<!-- Event Cost -->
```
Comment (borkweb) [inline src/views/list/single-event.php]: You'll need to update the `@version` line in this file's docblock at the top of the file

File: src/views/day/single-event.php (hunk 1 above)
Comment (jesseeproductions) [inline src/views/day/single-event.php]: Added

File: src/views/list/single-event.php (hunk 2 above)
Comment (jesseeproductions) [inline src/views/list/single-event.php]: Already there :)

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:55](https://github.com/moderntribe/the-events-calendar/blob/de91fa6273c1798286c8f15d6dd1ac1c736b811d/src/views/day/single-event.php#L55)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/de91fa6273c1798286c8f15d6dd1ac1c736b811d/src/views/list/single-event.php#L64)

> via jenkins/codesniffer-phpcs

**Comment by borkweb** [comment]
:+1:

**Pull request merged**